.github/workflows/pipeline.yml

```diff
@@ -34,6 +34,12 @@ jobs:
       environment: 'development'
       app_version: '${{ needs.docker_build.outputs.app_version }}'
 
+  cas3_generate_types:
+    name: Run the generate types job for the CAS3 UI
+    needs: deploy_dev
+    uses: ministryofjustice/hmpps-temporary-accommodation-ui/.github/workflows/generate-types.yml@main
+    secrets: inherit
+
```

Comment:
Move this after e2e to ensure endpoint is available, and ensure `needs` is set to `e2e`